docs/targets/confluent.md

```diff
@@ -1,60 +1,59 @@
 
 # Event Target for Confluent
 
-This event target receives [CloudEvents][ce] and forwards the event to a Confluent Kafka cluster.
+This event Target receives [CloudEvents][ce] and forwards the event to a Confluent Kafka cluster.
 
-## Prerequisites
+## Prerequisite(s)
 
-1. Access to a Kafka cluster with appropriate configuration details.
+- Access to a Kafka cluster with appropriate configuration details
 
-If a password is required, then consult the [Secrets](../guides/secrets.md) guide
-for additional information about how to add the password as a secret.
+If a password is required, consult the [Secrets](../guides/secrets.md) guide
+for additional information on how to add the password as a secret.
 
----
+## Kafka cluster details
 
-**NOTE**: Depending on the cluster and user permissions, the Kafka topic must
-exist prior to setting up the target. Otherwise, the target will attempt to create
+Depending on the cluster and user permissions, the Kafka topic must
+exist prior to setting up the target. Otherwise, the Target will attempt to create
 the topic which will require setting the `Topic replication` and `Topic partition`.
 
----
-
 ## Deploying an Instance of the Target
 
 Open the Bridge creation screen and add a Target of type `Confluent`.
 
 ![Adding a Confluent Target](../images/confluent-target/create-bridge-1.png)
 
-In the Target creation form, provide a name to the event Target, and add the following information:
+In the Target creation form, provide a name for the event Target and add the following information:
 
-* **Password Secret**: Reference to a [TriggerMesh secret](../guides/secrets.md) containing the password associated with the user accessing the Kafka cluster as described in the Prerequisites.
-* **Bootstrap Servers**: Confluent bootstrap servers to connect to. Use `ADD MORE` button to add additional bootstrap servers if needed.
+* **Password Secret**: Reference to a [TriggerMesh secret](../guides/secrets.md) containing the password associated with the user accessing the Kafka cluster as discussed in the prerequisites.
```

Comment:
What do you think about making prerequisites a link? Once it is clicked, it will scroll to the top to the Prerequisites spot on the page? If you think this will be good to do, I can help you with the syntax.

Reply:
I think that would be a nice detail. I'll add that and let you know if I have trouble with the syntax. 👍